### PR TITLE
Cp/network details page

### LINKED ISSUE #10: [Pages] Network Status & Details
# Context

The following pages are required:
- [Network Status](https://www.figma.com/proto/jxzg0xbPBLbWz2BFsFPPx1/Prototypes?node-id=1654%3A27090&scaling=min-zoom&page-id=1429%3A21963&starting-point-node-id=1654%3A22500)
- [Network Details](https://www.figma.com/proto/jxzg0xbPBLbWz2BFsFPPx1/Prototypes?node-id=1654%3A27338&scaling=min-zoom&page-id=1429%3A21963&starting-point-node-id=1654%3A22500)

# Acceptance Criteria

The following components can be created from the Network Status page:
- [x] Uses the appropriate layout (landing page layout) to display the components from https://github.com/onflow/next-docs-v1/issues/17 
- [x] Has a callout card and footer once components are available
- [x] Clicking on a network takes you to the corresponding tab in the network details page.

The following components can be created from the Network Details page:
- [x] Uses the appropriate layout (landing page layout) to display the components from https://github.com/onflow/next-docs-v1/issues/17 
- [x] Tabs have individual routes that are mapped from the network status page and switch the content below on click
- [x] Certain fields with the copy icon should copy to the clipboard in the spork card details
- [x] Past Spork Card & List are paginated

### PR DESCRIPTION
**Pull request by aureentuluvacal**
Closes #10

### CODE REVIEW

**Review by 10thfloor — CHANGES_REQUESTED** [review]
Hey what about the background gradients?

File: libs/design-system/src/lib/Components/SporksCard/index.tsx [hunk 1]
```diff
@@ -49,24 +51,32 @@ const Spork = ({ heading, timestamp, sporkMetadata }) => {
     branchOrTag,
     dockerTag,
   } = sporkMetadata;
-  const [isExpanded, setIsExpanded] = useState(true);
+  const [isExpanded, setIsExpanded] = useState(isDefaultExpanded);
+  const cardStyles = clsx(
```
Comment (10thfloor) [inline libs/design-system/src/lib/Components/SporksCard/index.tsx:55]: Sometimes we use `clsx` sometimes we don't? What is the rule for deciding when to use and when not to use?

File: libs/design-system/src/lib/Pages/NetworkDetailPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+import {
+  Footer,
+  FeaturedArticleSlider,
+  SocialLinksSignup,
+  Callout,
+  TabMenu,
+  NetworkDetailsCard,
+  Pagination,
+} from '../../Components';
+import SporksCard from '../../Components/SporksCard';
```
Comment (10thfloor) [inline libs/design-system/src/lib/Pages/NetworkDetailPage/index.tsx:12]: Can this import be included in the above de-structured imports?

File: libs/design-system/src/lib/Pages/NetworkDetailPage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+            branchOrTag: 'v0.22.9-patch-1-epoch-view-check-hotfix',
+            dockerTag: 'v0.22.9-patch-1-epoch-view-check-hotfix',
+          }}
+        />
+        {[1, 2, 3, 4].map((_) => (
+          <div className="my-6">
+            <SporksCard
+              heading={currentNetwork.name}
+              timestamp={endOfWeek(new Date())}
+              sporkMetadata={{
```
Comment (10thfloor) [inline libs/design-system/src/lib/Pages/NetworkDetailPage/index.tsx:109]: We should add an interface type for this component.

**Review by 10thfloor — CHANGES_REQUESTED** [review]

**Comment by aureentuluvacal** [comment]
> Hey what about the background gradients?

The network details page only had the blue/green gradients for the social links in the designs.

File: libs/design-system/src/lib/Components/NetworkDetailsCard/index.tsx [hunk 4]
```diff
@@ -56,11 +58,13 @@ const NetworkDetailsCard = ({
...
       <div className="px-8 py-4">
-        <p className="mb-2 text-xs uppercase text-primary-gray-200">RSS Feed</p>
+        <p className="mb-2 text-xs uppercase dark:text-primary-blue-dark text-primary-gray-200">
+          RSS Feed
+        </p>
         {/* @ts-ignore */}
         <Link
           href={rssFeed}
-          className="flex items-center text-sm text-primary-blue hover:cursor-pointer dark:text-primary-gray-50"
+          className="flex items-center text-sm hover:cursor-pointer"
```
Comment (alse) [inline libs/design-system/src/lib/Components/NetworkDetailsCard/index.tsx:67]:
```suggestion
          className="flex items-center text-sm"
```

Links are still a bit of a work in progress. Non-external links require passing the <a href... tag as part of its children. I think the external ones will be updated to work the same way. cc @10thfloor

**Review by alse — APPROVED** [review]

File: libs/design-system/src/lib/Components/TabMenu/index.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        const indicatorClasses = clsx(
+          'bg-black rounded-tr-lg rounded-tl-lg h-2 mt-4 dark:bg-white',
+          { block: isCurrentIndex, hidden: !isCurrentIndex }
+        );
+
+        return (
+          <div key={tab} className="min-w-[6rem] text-center md:min-w-[10rem]">
+            <span
+              className={classes}
+              onClick={() => {
```
[the remark this answers is not in the document]
Comment (10thfloor) [reply libs/design-system/src/lib/Components/TabMenu/index.tsx]: Yes. We'll use Remix nested routes with the `<Outlet/>`

[the remark this answers is not in the document]
Comment (10thfloor) [reply libs/design-system/src/lib/Components/TabMenu/index.tsx]: We may have to refactor these depending on how we decide to manage the layouts in the Remix app.

**Pull request merged**